Material.Icons.Avalonia/MaterialIconText.axaml.cs

```diff
@@ -95,17 +97,18 @@ public VerticalAlignment VerticalContentAlignment {
         protected override void OnApplyTemplate(TemplateAppliedEventArgs e) {
             base.OnApplyTemplate(e);
 
-            var classes = Classes;
-            if (classes.Count > 0) {
-
-                // Redirect classses to the template parts
+            // Ignore pseudo-classes replication, otherwise crash with: The pseudoclass ':xxx' may only be added by the control itself.
+            var classes = Classes.Where(s => s.Length > 0 && s[0] != ':');
+            var filteredClasses = classes as string[] ?? classes.ToArray();
+            if (filteredClasses.Length > 0) {
```

Comment:
No need to to `Where` and array creation if classes are empty

Move the code under the if